src/clean_split_data.py

```diff
@@ -0,0 +1,47 @@
+# author: MDS-2021-22 block3 group21
+# date: 2021-11-25
+"""
+Cleans NCDB 2017 data and creates training and test data set csv files.
+
+Usage: clean_split_data.py --input=<input filepath> --output=<output directory>
+
+Options:
+--input=<input filepath>    Filepath of data in csv format
+--output=<output directory> Directory specifying where to store training and test data sets
+"""  
+
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from docopt import docopt
+
+opt = docopt(__doc__)
+
+def main():    
+    # Importing NCDB 2017 dataset and dropping irrelevant or redundant columns
+    ncdb = (
+        pd.read_csv(opt["--input"], low_memory=False)
+        .sort_index()
+        .drop(columns=["C_YEAR", "C_CASE", "C_SEV"])
```

Comment:
Year is always 2017, so it is irrelevant, CASE is like an ID column which is also irrelevant, and SEV contains the same info as P_ISEV so it's redundant. However, looking at the data dictionary now, C_SEV has only values of 1: fatality and 2:non-fatality, so it probably would've made more sense to use this as the target

Reply:
I did notice that while doing EDA - C_SEV is on collision level while P_ISEV is on personal level. If we just care about whether a collision causes at least one fatality, definitely we can use C_SEV as target column instead of creating the additional FATALITY column.